cvat/apps/iam/permissions.py

```diff
@@ -126,7 +133,8 @@ def __init__(self, **kwargs):
                 'auth': {
                     'user': {
                         'id': self.user_id,
-                        'privilege': self.group_name
+                        'privilege': self.group_name,
+                        'has_analytics_access': self.has_analytics_access,
```

Comment:
Looks like these changes should be encapsulated in `LogViewerPermission`.

Reply:
done